api/envoy/config/core/v3/protocol.proto

```diff
@@ -674,6 +674,10 @@ message Http3ProtocolOptions {
   // docs](https://github.com/envoyproxy/envoy/blob/main/source/docs/h2_metadata.md) for more
   // information.
   bool allow_metadata = 6;
+
+  // Disables QPACK compression related features for HTTP/3 including:
+  // No huffman encoding, zero dynamic table capacity and no cookie crumbing.
+  bool disable_qpack = 7;
```

Comment:
You'll need to update the the changelog to mention this new API `changelogs/current.yaml`.

Reply:
It would also be nice to mention why someone might want to do this?